action/selection/open_spiel/database.cc

```diff
@@ -220,9 +222,22 @@ struct ExplainMicroserviceCost {
           std::cerr << absl::StrCat("ERROR bad result: status ", res->status, " url ", url, " body ", res->body)
                     << std::endl;
         }
+
+        if (record_results)
+        {
+          // If we're recording results, then append this particular inference result to the accumulator.
+          inference_results_ << (res->body.c_str()) << std::endl;
+        }
       } else {
         std::cerr << absl::StrCat("ERROR unknown: status ", res->status, " url ", url, " body ", res->body)
                   << std::endl;
+
+        if (record_results)
+        {
+          // If we're recording results, then append this particular inference result to the accumulator.
+          // We'll be able to identify this failure since the model_cost_ will show a low prediction.
+          inference_results_ << (res->body.c_str()) << std::endl;
```

Comment:
Since the `if` and `else` code paths are identical, move it to after the if (with your documentation)?